service-api/src/main/java/greencity/constant/ErrorMessage.java

```diff
@@ -89,6 +89,8 @@
     public static final String SOME_CERTIFICATES_ARE_INVALID =
         "SOME CERTIFICATES ARE INVALID. A valid certificate is listed here";
     public static final String CERTIFICATE_NOT_FOUND = "CERTIFICATE_NOT_FOUND";
+    public static final String REGION_IS_NOT_FOUND = "Region with chosen name doesn't found ";
```

Comment:
`doesn't found ` is not grammatically correct. Use: `Region with chosen name doesn't exist`, for example